### ISSUE TITLE
A11y_AzureDataStudio_ManageAccessForFile_Keyboard : Keyboard focus is not  landing on the form as soon as we click on manage access. 

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7794]
GitHub Tags:#A11y_ADS_OctTestPass;#A11yMAS; #A11yTCS;#SQL 
AzureDataStudio;#MAS2.4.3;#DesktopApp;#Mac;#Keyboard;

**Environment Details: **

Application Name: Azure Data Studio

Version: 1.12.0-insider (user setup)

Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7

Date: 2019-10-03T08:56:27.454Z

VS Code: 1.38.0

Electron: 4.2.10

Chrome: 69.0.3497.128

Node.js: 10.11.0

V8: 6.9.427.31-electron.0

OS: MacOS Mojave 10.14.6

 

**Additional Details:**

Tool Name :Voice Over
**Steps to Reproduce:**

1. Install and Launch the application the azure data studio application.

2.Connet to the servers with the required credentials.

3.Navigate to the control panel and click on the “Jupyter books” icon. 

 4.Navigate till HDFS folder and right click on it and click on manage access.

5. Navigate with the help of keyboard "tab"

7.Observe the focus on the screen.

 

**Actual:**
Keyboard focus is not landing on the form as soon as we click on manage access.
This issue is through out the application.

 

**Expected:** 
Keyboard focus is should landing on the form as soon as we click on manage access.
 

**User Impact:**
Key board user will be confused if the focus remains out of the screen.
 

**Attachment for Reference:*

[7794_A11y_AzureDataStudio_ManageAccessForFile_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3760964/7794_A11y_AzureDataStudio_ManageAccessForFile_ScreenReader.pptx)

**Comment by vinodbk197** [comment]
@Charles-Gagnon 
The issue is still reproduces, As soon as "ManageAccessForFile" screen get opened by the next tab press  the keyboard focus should present on the "ManageAccessForFile" screen for the first actionable control.
[7794_Reopened.pptx](https://github.com/microsoft/azuredatastudio/files/4139253/7794_Reopened.pptx)

**Comment by vinodbk197** [comment]
@Charles-Gagnon 
Verified this issue on :Version: 1.15.0-insider (user setup)
The issue is still reproduced, Please refer the above comments. we are not able to reopen the bug. Kindly reactivate the bug as this bug is still reproduced.

**Comment by chlafreniere** [comment]
Should be fixed by #9114.

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_Mac;

**Comment by sspatil123** [comment]
#A11ySev2;
